style.css

```diff
@@ -0,0 +1,117 @@
+.flex-row {
+  display: flex;
+  flex-direction: row;
+}
+
+.flex-column {
+  display: flex;
+  flex-direction: column;
+}
+
+.container-header {
+  justify-content: space-between;
+  align-items: center;
+  width: 375px;
+  height: 92px;
+  background: #3c3a39;
+}
```

Comment:
Kindly update your header and headline sections to match the figma design better🙏

Your design:
<img width="251" alt="image" src="https://user-images.githubusercontent.com/37116322/235757009-9f1ffe0a-edde-48c9-87fc-90d8283106a8.png">

Figma design:
<img width="201" alt="image" src="https://user-images.githubusercontent.com/37116322/235757460-4463d625-2752-4a6a-9652-054cdb7d2b13.png">

A few things to note:
- There is currently a lot of white space around your design. To fix this, try adding `* { box-sizing: border-box; }` and `body {
  margin: 0;
  padding: 0;
}` at the top of your CSS file. This will help to remove any default margin in your elements
- Afterwards, you can try adding `outline: 1px solid black` in the `*` style rule to help see which elements are causing  the horizontal whitespace. When you identify them, please update the styles so that there is no extra whitespace in your design